samples/python/kalman.py

```diff
@@ -81,12 +81,12 @@ def draw_cross(center, color, d):
 
             kalman.correct(measurement)
 
-            process_noise = kalman.processNoiseCov * np.random.randn(2, 1)
+            process_noise = sqrt(kalman.processNoiseCov[0,0]) * np.random.randn(2, 1)
             state = np.dot(kalman.transitionMatrix, state) + process_noise
 
             cv2.imshow("Kalman", img)
 
-            code = cv2.waitKey(100) % 0x100
+            code = cv2.waitKey(100)
```

Comment:
Yes. Thanks. Quick searching says it could be strange especially for non-letter keys. Made one more fix.
```
 if (code % 0x100) in [27, ord('q'), ord('Q')]:
```